### DISCUSSION TITLE
How to correctly quit an app?

### DISCUSSION THREAD

**Discussion by enbugger** [bevyengine/bevy#2745] [Q&A] [answered]
Would `std::process::exit` be enough for this?

**Comment by DJMcNab** [comment, marked as answer]
You should be able to send an `AppExit` event, i.e.

```rust
fn exit_app_system(mut writer: EventWriter<AppExit>){
    writer.send(AppExit);
}
```

**Reply by enbugger** [reply]
Thanks, works. `writer` should be `mut` afaiu.

**Reply by DJMcNab** [reply]
Ah well caught - updated

That's why we need more examples in the documentation :P.
